cli/copyFeatures.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+
+
+/**
+ * Copies all '*.feature' files from the 'srcDir' directory to the 'dstDir'
+ * directory, allowing for a transformation of the file basename and
+ * the content on the way using the 'transform()' function
+ */
+module.exports = function copyFeatures(srcDir, dstDir, transform) {
+  glob.sync(path.join(srcDir, '*.feature')).forEach((featureSrcPath) => {
+    const featureSrcContent = fs.readFileSync(featureSrcPath, { encoding: 'utf-8' });
```

Comment:
Should we handle if the reading/writing fails?
For example, using these methods as promises, and broadcasting the output to the process?

Reply:
I don't expect reading or writing issues in this case. If they happen, I believe the sync fs should throw, which crashes the CLI with an error, very likely a quite comprehensive one (e.g. permission read error or something like that, I suppose).